onnx_coreml/_operators_nd.py

```diff
@@ -49,9 +150,211 @@ def _convert_matmul(builder, node, graph, err):
                                     output_name=node.outputs[0])
 
 
+def _convert_reshape(builder, node, graph, err):
+    '''
+    convert to CoreML Reshape Static Layer:
+    https://github.com/apple/coremltools/blob/655b3be5cc0d42c3c4fa49f0f0e4a93a26b3e492/mlmodel/format/NeuralNetwork.proto#L4844
+    '''
+    shape_node = node.inputs[1]
+    if shape_node in node.input_tensors:
+        output_shape = node.input_tensors[shape_node]
+    
+        # if rank is same, then call rank preserving reshape
+        if len(output_shape) == len(graph.shape_dict[node.inputs[0]]):
```

Comment:
need some error catching here, if ``node.inputs[0]`` is not in the ``shape_dict``